fossunited/foss_hackathon/doctype/foss_hackathon_localhost/foss_hackathon_localhost.py

```diff
@@ -24,14 +27,20 @@ class FOSSHackathonLocalHost(Document):
         )
 
         city: DF.Link | None
+        email: DF.Data | None
+        image: DF.AttachImage | None
         is_accepting_attendees: DF.Check
+        is_published: DF.Check
         localhost_name: DF.Data
         location: DF.Data | None
         map_link: DF.Data | None
         organizers: DF.Table[FOSSHackathonLocalHostOrganizer]
         parent_hackathon: DF.Link
+        route: DF.Data | None
+        slug: DF.Data | None
```

Comment:
why not make slug mandatory? that'll be better than "scrubbing" and creating it ourselves